### ISSUE TITLE
Lack of documentation

### ISSUE THREAD

**Issue by ehuff700** [harlanc/xiu#37]
Am I crazy or is there a total lack of documentation for the rtmp module? I would like to extend the functionality of that module and use it within my program but can't find any documentation for it anywhere.

**Comment by kaitlynia** [comment]
I agree there should be more documentation. There are examples of config files on the docs website, but not much information about the options within them. Also, the "Quick Start" link on the homepage of the docs website 404s.

**Comment by harlanc** [comment]
@kaitlynia  Please try the "Quick Start" link again to see if it's fixed or not
